src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/DuplexClientBaseTests.cs

```diff
@@ -11,10 +11,24 @@
 
 public class DuplexClientBaseTests : ConditionalWcfTest
 {
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
+#else
     [ConditionalFact(nameof(Root_Certificate_Installed))]
+#endif
     [OuterLoop]
     public static void DuplexClientBaseOfT_OverHttp_Call_Throws_InvalidOperation()
```

Comment:
Because it throws on "CreateChannel" it looks to me like this is the expected behavior and there is no positive scenario for using duplex client base with http?

@iamjasonp since this looks like his test.